packages/grant-explorer/src/features/discovery/RoundsSubNav.tsx

```diff
@@ -0,0 +1,124 @@
+  },
+  {
+    id: "0x59d79b22595b17af659ce9b03907615f53742c57",
+    name: "Education, Community Growth & Events Round",
+    chainId: ChainId.ARBITRUM,
+  },
+  {
+    id: "0x8dce7a66e0c310f9f89e847dba83b2344d589161",
+    name: "Vault Round 1",
+    chainId: ChainId.FANTOM_MAINNET_CHAIN_ID,
+  },
+];
+
+export function RoundsSubNav(props: Props) {
+  const [open, setOpen] = useState(false);
+  return (
+    <div className="bg-grey-150">
+      <div className="mx-auto px-4 sm:px-6 lg:px-20">
+        <div className="flex justify-between h-12 flex-row-reverse">
+          <div className="flex items-center gap-6">
+            <div>
+              <a
+                href="https://bridge.gitcoin.co"
+                target="_blank"
+                rel="noreferrer"
+                className="flex-shrink-0 flex items-center"
+              >
+                <PGNLogoIcon className="mr-2" />
+                <PGNTextLogoIcon fill="black" className="mt-1" />
+              </a>
+            </div>
+            <div className="relative bg-white">
+              <span
+                style={{
+                  border: "1px solid var(--sky-600, #15B8DC)",
+                  borderRadius: "8px",
+                  padding: "4px 8px",
+                  boxShadow:
+                    "0px 1px 2px 0px rgba(0, 0, 0, 0.06), 0px 1px 3px 0px rgba(0, 0, 0, 0.10)",
+                }}
+                className="cursor-pointer flex gap-2 items-center"
+                aria-label={"Open Grants Subnav"}
+                onClick={() => setOpen(!open)}
+              >
+                Active rounds
+                <DropdownIcon
+                  className="inline"
+                  direction={open ? "up" : "down"}
+                />
+              </span>
+              <div
+                className={`absolute right-0 top-12 bg-grey-150 py-4 px-6 rounded-lg text-right whitespace-nowrap ${
+                  open ? "block" : "hidden"
+                }`}
+              >
+                <ul>
+                  {rounds.map((round) => (
+                    <li key={round.id}>
+                      <a
+                        data-testid={`round-link-${round.id}`}
+                        href="#"
+                        onClick={(e: React.MouseEvent<HTMLAnchorElement>) => {
+                          e.preventDefault();
+                          props.onClick(round);
+                        }}
+                        className="flex justify-end py-2 items-center"
+                      >
+                        <img
+                          src={(() => {
+                            switch (round.chainId) {
+                              case ChainId.ARBITRUM:
+                                return "/logos/arb-logo.svg";
+                              case ChainId.FANTOM_MAINNET_CHAIN_ID:
+                                return "/logos/fantom-logo.svg";
+                            }
+                          })()}
```

Comment:
If you use the ternary here, you avoid the odd-looking inline call and force yourself to make all outcomes explicit:

```ts
src={
  round.chainId === ChainId.ARBITRUM
      ? "/logos/arb-logo.svg"
      : round.chainId === ChainId.FANTOM_MAINNET_CHAIN_ID
      ? "/logos/fantom-logo.svg"
      : null;
}
```